spec/models/group_assignment_invitation_spec.rb

```diff
@@ -58,19 +58,7 @@
 
     subject { create(:group_assignment_invitation, group_assignment: group_assignment) }
 
-    after(:each) do
-      RepoAccess.destroy_all
-      Group.destroy_all
-      GroupAssignmentRepo.destroy_all
-      GroupInviteStatus.destroy_all
-    end
-
     context "success result" do
-      it "success?" do
```

Comment:
This test can be removed since `#redeem_for` no longer returns `success` when a assignment repo doesn't yet exist. Instead, after the `group_import_resiliency` it returns `pending`.